icons/banknote-arrow-down.json

```
@@ -0,0 +1,19 @@
+{
+  "$schema": "../icon.schema.json",
+  "contributors": [
+    "AnnaSasDev",
+    "joffx"
+  ],
+  "tags": [
+    "money",
+    "cash",
+    "payment",
+    "finance",
+    "arrow",
+    "growth",
+    "increase"
```

Tags from the original icon should also be present, as well as tags covering use cases suggested by @AnnaSasDev. Arrow as a tag is not needed, since it's part of the name. Additionally, you've mixed up tags between  `-arrow-up` and `-arrow-down`.

```suggestion
    "bill",
    "currency",
    "money",
    "payment",
    "funds",
    "transaction",
    "cash",
    "finance",
    "withdraw",
    "expense",
    "out",
    "payout",
    "refund",
    "debit",
    "spending",
    "decrease"
```